src/ecdsa/keys.py

```diff
@@ -981,7 +1049,7 @@ def to_string(self):
         s = number_to_string(secexp, self.privkey.order)
         return s
 
-    def to_pem(self, point_encoding="uncompressed"):
+    def to_pem(self, point_encoding="uncompressed", format="sslay"):
```

Comment:
sorry, it looks like I made a typo when documenting the methods, the format is "ssleay", from the predecessor of openssl: https://en.wikipedia.org/wiki/SSLeay